x/evm/watcher/watcher.go

```diff
@@ -412,6 +413,12 @@ func (w *Watcher) commitBatch(batch []WatchMessage) {
 			w.store.Delete(key)
 		} else {
 			w.store.Set(key, value)
+			//need update params
+			if bytes.Compare(key, prefixParams) == 0 {
```

Comment:
Maybe we can expand a type for params, so we do not need to always compare bytes here.